lxd/db/cluster/identities.go

```diff
@@ -447,7 +448,7 @@ func GetPendingTLSIdentityByTokenSecret(ctx context.Context, tx *sql.Tx, secret
 	// Convert identity types to strings for the IN clause.
 	typeStrings := make([]string, len(identityTypes))
 	for i, idType := range identityTypes {
-		typeStrings[i] = fmt.Sprintf("%d", idType)
+		typeStrings[i] = strconv.FormatInt(idType, 10)
```

Comment:
This probably isn't needed any more (see earlier comment). In any case, generally when I'm adding new code I'll fix lint errors in the addition rather than adding code and fixing linter errors in a separate commit.